src/browser/modules/Editor/Editor.jsx

```diff
@@ -499,7 +501,12 @@ export class Editor extends Component {
     return (
       <Bar>
         <Header>
-          <ActionButtons width={16} buttons={buttons} />
+          <ActionButtons
+            width={16}
+            buttons={buttons}
+            editorValue={() => this.getEditorValue()}
+            isRelateAvailable={this.props.isRelateAvailable}
```

Comment:
in order to get tests to pass. I will look at additional tests during code freeze.